tests/basic-test.sh

```diff
@@ -450,16 +450,20 @@ echo sudoers2 >tree-C/etc/sudoers
 $OSTREE commit ${COMMIT_ARGS} -b test3-C2 -s "C" --tree=dir=tree-C
 echo sudoers1 >tree-C/etc/sudoers
 
-$OSTREE commit ${COMMIT_ARGS} -b test3-mix1 -s "A" --tree=ref=test3-C1 --tree=ref=test3-D
-$OSTREE commit ${COMMIT_ARGS} -b test3-mix2 -s "A" --tree=dir=tree-C --tree=ref=test3-D
-$OSTREE commit ${COMMIT_ARGS} -b test3-mix3 -s "A" --tree=ref=test3-C1 --tree=dir=tree-D
-$OSTREE commit ${COMMIT_ARGS} -b test3-mix4 -s "A" --tree=dir=tree-C --tree=dir=tree-D
+$OSTREE commit ${COMMIT_ARGS} -b test3-ref-ref -s "A" --tree=ref=test3-C1 --tree=ref=test3-D
+$OSTREE commit ${COMMIT_ARGS} -b test3-dir-ref -s "A" --tree=dir=tree-C --tree=ref=test3-D
+$OSTREE commit ${COMMIT_ARGS} -b test3-ref-dir -s "A" --tree=ref=test3-C1 --tree=dir=tree-D
+$OSTREE commit ${COMMIT_ARGS} -b test3-dir-dir -s "A" --tree=dir=tree-C --tree=dir=tree-D
```

Comment:
Minor/optional: the `-s` switch is optional nowadays.

Reply:
Fixed in #1645